resources/roadmap.json

```diff
@@ -19,5 +19,12 @@
     "categories": [
       "INCOME_TAX_MTD"
     ]
+  },
+  {
+    "name": "UK Internal Market Authorisation Checker roadmap",
+    "context": "/guides/ukim-auth-checker-api-roadmap/",
```

Comment:
Is there a reason why the context isn't `/roadmap` instead of `/guides`?

Reply:
Because that's what the context was set up in https://jira.tools.tax.service.gov.uk/browse/SUP-14151